bridge-impl/src/main/java/com/liferay/faces/bridge/renderkit/bridge/internal/ResponseWriterBridgeCompat_2_0_Impl.java

```diff
@@ -48,6 +48,35 @@
 	protected static final String VIEW_STATE_MARKER = PartialResponseWriter.VIEW_STATE_MARKER;
 	protected static final String XML_MARKER = "<?xml";
 
+	// Private Constants
+	private static final boolean JSF_RUNTIME_SUPPORTS_NAMESPACING_VIEWSTATE;
+
+	static {
+
+		boolean namespacedViewStateSupported = false;
```

Comment:
Thanks for pointing that out. Corrected for Mojarra 2.3+ in subsequent commit 654542db4e140f5249f8a4ec699c65a54f796112